pkg/diff/sql_generator.go

```diff
@@ -1718,6 +1718,7 @@ func (isg *indexSQLVertexGenerator) addDepsOnTableAddAlterIfNecessary(index sche
 		}
 	}
 
+	// Otherwise, we can drop the index whenever we want.
```

Comment:
Initially was debugging this problem and thought adding a comment helped document it a little better